### PR TITLE
Multiple emails

### PR DESCRIPTION
**Pull request by gustavo-salazar**
Adds support to use multiple emails when sending notifications. 
Two methods are supported:
* `from_email` can be a comma separated list of emails. This generates RT tickets with multiple requestors. Validation included in `is_valid`.
* An optional field `cc` is now supported. Also comma separated, in case of multiple emails. and uses the same validation.

We will be using the first method, because the RT system doesn't send emails to CC when creating the ticket.

### CODE REVIEW

File: emgena/serializers.py [hunk 1]
```diff
@@ -84,6 +98,8 @@ def create(self, validated_data):
             "Subject": n.subject,
             "Text": n.message.replace("\n", ';')
         }
+        if n.cc != '' and n.cc is not None:
```
Comment (SandyRogers) [inline emgena/serializers.py:101]:
```suggestion
        if n.cc:
```
(`''` and `None` are both falsy)`

File: emgena/serializers.py [hunk 2]
```diff
@@ -58,12 +58,26 @@ class Meta:
...
+    cc = serializers.CharField(max_length=200, allow_blank=True, required=False)
     subject = serializers.CharField(max_length=500, required=True)
     message = serializers.CharField(max_length=1000, required=True)
     is_consent = serializers.BooleanField(default=False, required=False)
 
+    def is_valid(self):
+        for field in ['from_email', 'cc']:
+            if field not in self.initial_data:
+                continue
+            for email in self.initial_data[field].split(','):
```
Comment (SandyRogers) [inline emgena/serializers.py:71]: Is it worth doing a `email = email.strip()` here just in case there is leading/trailing whitespace?

**Review by SandyRogers — COMMENTED** [review]
Looks good, nice work. Made some small inline comments. 

I suppose you considered and decided against the alternative of having a plural `from_emails` field with `many=True` instead; and just (usually) sending a 1-length array of `from_emails: ["tim@apple.com"]`?

File: emgena/serializers.py (hunk 1 above)
Comment (gustavo-salazar) [reply emgena/serializers.py:101]: TIL you can do that in python!
Thanks

File: emgena/serializers.py (hunk 2 above)
Comment (gustavo-salazar) [reply emgena/serializers.py:71]: Sure it is. I know I think about it but I guess I forgot. I'll commit a change with it

**Comment by gustavo-salazar** [comment]
> Looks good, nice work. Made some small inline comments.
> 
> I suppose you considered and decided against the alternative of having a plural `from_emails` field with `many=True` instead; and just (usually) sending a 1-length array of `from_emails: ["tim@apple.com"]`?

I did consider the idea of an array of emails, but that might affect the API of the endpoint, and I wanted to avoid changes at that level. So I though a comma separated string is the simplest, specially if at the end that's the needed format when sending it to RT. ON the other hand, I had to add the custom validator for it instead of the ones on ListField andMailField. 🤷

**Comment by SandyRogers** [comment]
> I did consider the idea of an array of emails, but that might affect the API of the endpoint, and I wanted to avoid changes at that level. So I though a comma separated string is the simplest, specially if at the end that's the needed format when sending it to RT. ON the other hand, I had to add the custom validator for it instead of the ones on ListField andMailField. 🤷

Yeh, that makes sense. I thought I remembered some CSV Field type (which was still just a string field, but separated multiple entries if the given separator character `,` was present) – alas that was actually a `django-filters` thing, not serializers.

**Pull request merged**